### ISSUE TITLE
Error when searching trough database tables

### ISSUE THREAD

**Issue by faselunare** [wintercms/wn-builder-plugin#36]
I get this error when using the search table in the database view
now is it impossible to exit from this error
```
[2022-04-09 21:46:28] development.ERROR: TypeError: mb_strtolower(): Argument #1 ($string) must be of type string, array given in /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Support/Str.php:443
Stack trace:
#0 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Support/Str.php(443): mb_strtolower(Array, 'UTF-8')
#1 /Users/francesco/Dev/sites/wn-conference/modules/backend/Traits/SearchableWidget.php(36): Illuminate\Support\Str::lower(Array)
#2 /Users/francesco/Dev/sites/wn-conference/plugins/winter/builder/widgets/DatabaseTableList.php(93): Winter\Builder\Widgets\DatabaseTableList->textMatchesSearch(Array, Array)
#3 /Users/francesco/Dev/sites/wn-conference/plugins/winter/builder/widgets/DatabaseTableList.php(117): Winter\Builder\Widgets\DatabaseTableList->getData(Object(Winter\Builder\Classes\PluginVector))
#4 /Users/francesco/Dev/sites/wn-conference/plugins/winter/builder/widgets/DatabaseTableList.php(39): Winter\Builder\Widgets\DatabaseTableList->getRenderData()
#5 /Users/francesco/Dev/sites/wn-conference/plugins/winter/builder/controllers/index/_sidepanel.htm(11): Winter\Builder\Widgets\DatabaseTableList->render()
#6 /Users/francesco/Dev/sites/wn-conference/modules/system/Traits/ViewMaker.php(246): include('/Users/francesc...')
#7 /Users/francesco/Dev/sites/wn-conference/modules/system/Traits/ViewMaker.php(96): Backend\Classes\Controller->makeFileContents('/Users/francesc...', Array)
#8 /Users/francesco/Dev/sites/wn-conference/plugins/winter/builder/controllers/index/index.htm(5): Backend\Classes\Controller->makePartial('_sidepanel.htm')
#9 /Users/francesco/Dev/sites/wn-conference/modules/system/Traits/ViewMaker.php(246): include('/Users/francesc...')
#10 /Users/francesco/Dev/sites/wn-conference/modules/system/Traits/ViewMaker.php(108): Backend\Classes\Controller->makeFileContents('/Users/francesc...')
#11 /Users/francesco/Dev/sites/wn-conference/modules/backend/Classes/Controller.php(420): Backend\Classes\Controller->makeView('index')
#12 /Users/francesco/Dev/sites/wn-conference/modules/backend/Classes/Controller.php(297): Backend\Classes\Controller->execPageAction('index', Array)
#13 /Users/francesco/Dev/sites/wn-conference/modules/backend/Classes/BackendController.php(171): Backend\Classes\Controller->run('index', Array)
#14 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Routing/Controller.php(54): Backend\Classes\BackendController->run('winter/builder')
#15 /Users/francesco/Dev/sites/wn-conference/modules/backend/Classes/BackendController.php(119): Illuminate\Routing\Controller->callAction('run', Array)
#16 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Routing/ControllerDispatcher.php(45): Backend\Classes\BackendController->callAction('run', Array)
#17 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Routing/Route.php(261): Illuminate\Routing\ControllerDispatcher->dispatch(Object(Illuminate\Routing\Route), Object(Backend\Classes\BackendController), 'run')
#18 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Routing/Route.php(204): Illuminate\Routing\Route->runController()
#19 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Routing/Router.php(725): Illuminate\Routing\Route->run()
#20 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\Routing\Router->Illuminate\Routing\{closure}(Object(Illuminate\Http\Request))
#21 /Users/francesco/Dev/sites/wn-conference/modules/backend/Classes/BackendController.php(68): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#22 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(162): Backend\Classes\BackendController->Backend\Classes\{closure}(Object(Illuminate\Http\Request), Object(Closure))
#23 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Routing/Middleware/SubstituteBindings.php(50): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#24 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\Routing\Middleware\SubstituteBindings->handle(Object(Illuminate\Http\Request), Object(Closure))
#25 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/View/Middleware/ShareErrorsFromSession.php(49): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#26 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\View\Middleware\ShareErrorsFromSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#27 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(121): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#28 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Session/Middleware/StartSession.php(64): Illuminate\Session\Middleware\StartSession->handleStatefulRequest(Object(Illuminate\Http\Request), Object(Illuminate\Session\Store), Object(Closure))
#29 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\Session\Middleware\StartSession->handle(Object(Illuminate\Http\Request), Object(Closure))
#30 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/AddQueuedCookiesToResponse.php(37): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#31 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\Cookie\Middleware\AddQueuedCookiesToResponse->handle(Object(Illuminate\Http\Request), Object(Closure))
#32 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Cookie/Middleware/EncryptCookies.php(67): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#33 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Illuminate\Cookie\Middleware\EncryptCookies->handle(Object(Illuminate\Http\Request), Object(Closure))
#34 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#35 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Routing/Router.php(727): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#36 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Routing/Router.php(702): Illuminate\Routing\Router->runRouteWithinStack(Object(Illuminate\Routing\Route), Object(Illuminate\Http\Request))
#37 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Routing/Router.php(666): Illuminate\Routing\Router->runRoute(Object(Illuminate\Http\Request), Object(Illuminate\Routing\Route))
#38 /Users/francesco/Dev/sites/wn-conference/vendor/winter/storm/src/Router/CoreRouter.php(20): Illuminate\Routing\Router->dispatchToRoute(Object(Illuminate\Http\Request))
#39 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(167): Winter\Storm\Router\CoreRouter->dispatch(Object(Illuminate\Http\Request))
#40 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(141): Illuminate\Foundation\Http\Kernel->Illuminate\Foundation\Http\{closure}(Object(Illuminate\Http\Request))
#41 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Foundation/Http/Middleware/PreventRequestsDuringMaintenance.php(86): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#42 /Users/francesco/Dev/sites/wn-conference/vendor/winter/storm/src/Foundation/Http/Middleware/CheckForMaintenanceMode.php(25): Illuminate\Foundation\Http\Middleware\PreventRequestsDuringMaintenance->handle(Object(Illuminate\Http\Request), Object(Closure))
#43 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Winter\Storm\Foundation\Http\Middleware\CheckForMaintenanceMode->handle(Object(Illuminate\Http\Request), Object(Closure))
#44 /Users/francesco/Dev/sites/wn-conference/vendor/winter/storm/src/Foundation/Http/Middleware/CheckForTrustedProxies.php(56): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#45 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Winter\Storm\Foundation\Http\Middleware\CheckForTrustedProxies->handle(Object(Illuminate\Http\Request), Object(Closure))
#46 /Users/francesco/Dev/sites/wn-conference/vendor/winter/storm/src/Http/Middleware/TrustHosts.php(46): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#47 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(180): Winter\Storm\Http\Middleware\TrustHosts->handle(Object(Illuminate\Http\Request), Object(Closure))
#48 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Pipeline/Pipeline.php(116): Illuminate\Pipeline\Pipeline->Illuminate\Pipeline\{closure}(Object(Illuminate\Http\Request))
#49 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(142): Illuminate\Pipeline\Pipeline->then(Object(Closure))
#50 /Users/francesco/Dev/sites/wn-conference/vendor/laravel/framework/src/Illuminate/Foundation/Http/Kernel.php(111): Illuminate\Foundation\Http\Kernel->sendRequestThroughRouter(Object(Illuminate\Http\Request))
#51 /Users/francesco/Dev/sites/wn-conference/index.php(43): Illuminate\Foundation\Http\Kernel->handle(Object(Illuminate\Http\Request))
#52 /Users/francesco/.composer/vendor/laravel/valet/server.php(234): require('/Users/francesc...')
#53 {main}  
```

**Comment by faselunare** [comment]
fixed in #35 
but I have to add this code by hand...

how to install the latest version?
Thanks

**Comment by bennothommo** [comment]
@faselunare we're waiting on your response to https://github.com/wintercms/wn-builder-plugin/issues/32 before releasing a new version.
